aws/sdk/examples/rds-helloworld/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
```

Comment:
can you rename that file to just `rds` to match the other files?

Reply:
Yeah, I will update `rds.2014-10-31.json` tobe `rds.json`